src/unitStats/index.ts

```diff
@@ -1 +1,9 @@
+export * from "./faction";
 export * from "./squadTotalCost";
+export * from "./locstring";
+export * from "./mappingEbps";
+export * from "./mappingSbps";
+export * from "./mappingUpgrades";
+export * from "./mappingWeapon";
+export * from "./unitStatsLib";
+export * from "./weaponLib";
```

Comment:
Btw if we need all the files generally it's OK. But don't forget that it means that if you import the index --> it will import all the code in these files --> making the resulting code bigger than needed ( in case we don't need something) 

But I think that you use all of this right?

Reply:
Yeah, was wondering if tree-shaking was working out of the box. And yes, I am using all of these at least when loading the `getStaticProps` within the `[raceId].tsx` page.

I can rollback it if tree-shaking isn't enabled.